website/client/components/settings/api.vue

```diff
@@ -85,13 +85,28 @@ export default {
       showApiToken: false,
     };
   },
+  mounted () {
+    window.addEventListener('message', this.receiveMessage, false);
+  },
+  destroy () {
+    window.removeEventListener('message', this.receiveMessage);
+  },
   computed: {
     ...mapState({user: 'user.data', credentials: 'credentials'}),
     apiToken () {
       return this.credentials.API_TOKEN;
     },
   },
   methods: {
+    receiveMessage (eventFrom) {
+      if (event.origin !== 'https://www.spritely.app') return;
```

Comment:
should the origin also check for the url without the `www`?

Reply:
I think not, because www is forced for Spritely